inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -228,9 +228,10 @@ def recover(
             self._rollback_and_delete_snap()
             logger.debug("Rebooting to recover from failed SOTA...")
             rebooter.reboot()
-        else:            
+        else:
             if self._reboot_device:
-                logger.debug(f"Rebooting to recover from failed SOTA...time_to_wait_before_reboot = {str(time_to_wait_before_reboot)}")
+                logger.debug(
+                    f"Rebooting to recover from failed SOTA...time_to_wait_before_reboot = {str(time_to_wait_before_reboot)}")
```

Comment:
<h3>$\color{#FFA500}Improvements: CodeQuality$</h3>

Refactor the logging statement to use string formatting instead of concatenation for better performance and readability.

Reply:
out of scope--this is from reformatting the code base